app/config/config.yml

```diff
@@ -25,7 +25,6 @@ services:
     resource: "%kernel.root_dir%/../src/PrestaShopBundle/Controller/*"
     exclude: "%kernel.root_dir%/../src/PrestaShopBundle/Controller/Api"
     tags:
-      - 'controller.service_arguments'
```

Comment:
The change seems a bit risky indeed base on what @matks said regarding this initial change https://github.com/PrestaShop/PrestaShop/pull/18845

I see that even with your modification you still had to change the module decorator definition and add the tag again to the decorator again

So maybe this is actually what fixed the problem regardless of the core, does it work if you leave the core unchanged but add the tag in the module definition? Then it might mean that with Symfony 4.4 we now have to explicitly add this tag in the modules and we should "just" update our documentation.

Reply:
yeah ok, so I don't know what causes it (probably some changes in symfony core between versions). But it seems the tags conflicts in some way.
Only way it works if I remove the tag from general config AND explicitly add it in decorated service config.
For some reason if the tag exists in both places it doesn't work OR if the tag exists only in general config it doesn't work either.

Maybe it was introduced to symfony by this? :thinking: 
https://github.com/symfony/symfony/pull/30417/files

Reply:
@zuk3975 Confirmed.

Basically in Symfony 4.2, tags are removed from decorated controllers. But removing the tag from controllers break them so they are not "visible" by container. So from container point of view we have service A who aims to decorate service B, but service B does not exist.

😢